bundles/config/org.eclipse.smarthome.config.discovery/OSGI-INF/InboxAutoIgnore.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No, you don't - if you add a .gitinore for this specific generated XML into the OSGI-INF folder. Which is okay for that purpose until all are migrated. 

Migrating the others too just gives extra-kudos, but is not mandatory.